site/deleteresult.html.jet

```diff
@@ -0,0 +1,16 @@
+{{extends "templates/application/application.jet"}}
+
+{{block head()}}
+  {{yield seo(title=i18n("confirm_delete_result_page_header"), index=false)}}
+{{end}}
+
+{{block body()}}
+  <main id="main" class="page verify-email-page">
```

Comment:
feel a bit weird using a class for "verify email page" - just for maintainability we might change it not realising it also changes the delete result page. not sure on the best approach here, theres prolly a nicer way to merge the verify email and confirm delete styles into something more generic, whilst maintaining a unique selector if we do want to override things in the future. could prolly call this delete-confirm-page and just add .delete-confirm-page    as another class which uses the same styles from the stylesheet.

Reply:
Fair. Will give that a go.